src/Chapter2.hs

```diff
@@ -336,7 +348,10 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList x y l =
+    if (x <= y) && (x >= 0)
+       then (drop x (take (y+1) l))
```

Comment:
I would recommend adding spaces to both sides of binary operators :arrow_left: :space_invader: :arrow_right: 
```suggestion
       then (drop x (take (y + 1) l))
```